config/locales/client.en.yml

```
@@ -1515,6 +1515,8 @@ en:
       liked_many:
         one: "<span>{{username}}, {{username2}} and 1 other</span> {{description}}"
         other: "<span>{{username}}, {{username2}} and {{count}} others</span> {{description}}"
+      liked_consolidated_description: "liked {{number}} of your posts"
```

This needs proper pluralization:

```suggestion
      liked_consolidated_description:
        one: "liked {{count}} of your posts"
        other: "liked {{count}} of your posts"
```

---

The min for the site setting is 3 which is why  didn’t pluralize this.

---

I agree, it wouldn't be necessary for English. Unfortunately other languages have pluralization rules that are more complicated than the English ones.

---

eg, Russian:

Quantity ends in 1 - As per position in the sentence.
Quantity ends in 2,3,4 - Genitive Singular.
Quantity ends in 5,6,7,8,9,0,-надцать - Genitive Plural
General Quantity - Genitive Case (Singular or Plural depending on context.)
Quantity not specified - Use the case that is appropriate to the sentence position.

source: http://www.russianlessons.net/lessons/lesson11_main.php

---

Ahh icic. Thanks for clarifying, I'll make the changes.